test/test_rosbag/bag_migration_tests/README.md

```diff
@@ -0,0 +1,349 @@
+# Bag migration tests scripts
+
+In order to generate the data and rules for bag migration tests, two scripts are provided:
+
+ * **generate_data** : Generates *.bag* files with messages (data) to run the bag migration tests. 
+ * **generate_rules** : Help in the process of creating rules between two message generations.
+
+The next sections explain how to use these scripts in detail. The rest are auxiliary scripts that should NOT be used directly.
+
+Before running any of these scripts you need to build the **test_rosbag** package at least once in your workspace and `source` the environment workspace setup file. This can be done running the following from the root of the workspace:
+
+``` bash
+$ catkin_make
+$ source devel/setup.bash
```

Comment:
I honestly don't know which one is a better option. I prefer `catkin_make` because I can pass `--only-pkg-with-deps test_rosbag` to build it faster. I could update the `README.md` to do the same.

With `catkin_make_isolated` the `--only-pkg-with-deps test_rosbag` option needs `--install` and a quick test has failed for me because of missed dependencies (like `generate_messages`). I don't use `catkin_make_isolated` often, so I'm not very familiar with the environment setup I need to make it work properly.